backend/spec/controllers/enrollments/export_spec.rb

```diff
@@ -60,7 +60,7 @@
     end
 
     it "does not make database queries" do
-      expect { subject }.to make_database_queries(count: 3)
+      expect { subject }.to make_database_queries(count: 4)
```

Comment:
This is a performance regression. We must keep the number of database query to 3.